### PR TITLE
Pre-Waku: RSC compatibility

### PR DESCRIPTION
**Pull request by pmelab**
## Description of changes

Switch to updated versions of various packages and adapt to incoming changes triggered by react server components.

## Motivation and context

First iteration of switching to an RSC based framework.

## How has this been tested?

- [x] Manually
- [x] Unit tests
- [x] Integration tests

### CODE REVIEW

File: apps/website/src/layouts/index.tsx [hunk 1]
```diff
@@ -12,10 +12,14 @@ export default function Layout({
...
+      executors={[{ executor: drupalExecutor(`/graphql`) }]}
+    >
+      <OperationExecutorsProvider
+        executors={[{ executor: data, id: FrameQuery }]}
+      >
         <Frame>{children}</Frame>
-      </OperationExecutor>
-    </OperationExecutor>
+      </OperationExecutorsProvider>
+    </OperationExecutorsProvider>
```
Comment (Leksat) [inline apps/website/src/layouts/index.tsx:22]: Can we use a single `OperationExecutorsProvider` passing an array of executors?

File: apps/website/src/utils/locale.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// TODO: straight copy of the same file in the UI package
```
Comment (Leksat) [inline apps/website/src/utils/locale.ts:1]: Why the file from the UI package isn't used?

**Review by Leksat — APPROVED** [review]
Looks good!

Just few questions :)

File: apps/website/src/utils/locale.ts (hunk 2 above)
Comment (pmelab) [reply apps/website/src/utils/locale.ts:1]: I did not want to create an outlier-dependency between ui and website that is only needed for Gatsby.

**Pull request merged**